app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,70 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  before_action :signed_in_only!, only: %i[new create]
+
+  # GET /posts
+  def index
+    @posts = Post.all.order('created_at DESC')
+    @post = Post.new
+  end
```

Comment:
- [ ] there's no need for `@post` in here

Reply:
SOLVED